docker/config/linea-besu-sequencer/sequencer.config.toml

```diff
@@ -60,7 +60,7 @@ plugin-linea-tx-pool-simulation-check-api-enabled=false
 plugin-linea-tx-pool-simulation-check-p2p-enabled=false
 plugin-linea-max-block-calldata-size=109000
 plugin-linea-max-tx-calldata-size=60000
-plugin-linea-max-block-gas=30000000
+plugin-linea-max-block-gas=50000000
```

Comment:
Why increase from 30M to 50M here? What's the contect of increasing the effective block gas limit?

Reply:
>What's the context of increasing the effective block gas limit? To align with our roadmap for increasing block gas limit in the future?

Yes, that was the idea.